dali/pipeline/operators/op_schema.h

```diff
@@ -174,9 +174,8 @@ class DLL_PUBLIC OpSchema {
   /**
    * @brief Notes that sequences can be used with this op
    */
-  DLL_PUBLIC inline OpSchema& AllowSequences(bool needs_flattening) {
+  DLL_PUBLIC inline OpSchema& AllowSequences() {
```

Comment:
So we definitely are getting rid of flattening as an option?

Reply:
I think so. No other operator is using it and it was a temporary workaround anyway.